deckhand/engine/schema/v1_0/document_schema.py

```diff
@@ -69,7 +69,9 @@
                 'layerDefinition': {
                     'type': 'object',
                     'properties': {
-                        'layer': {'enum': ['global', 'region', 'site']},
+                        'layer': {'enum': ['global', 'global-network',
```

Comment:
Note this will be a run-time list based on the active LayeringPolicy document that has been loaded in Deckhand. Won't be able to validate against a schema-based enumeration.